src/globals.js

```diff
@@ -31,7 +31,7 @@
  *
  * @type {{}}
  */
```

Comment:
This change is unrelated to the task. Necessary overall, but unrelated.

Reply:
Sorry about that - I needed to change this to make things on my dev work, and I forgot to revert it.  Do you want me to revert & commit and publish this as its own PR?

Reply:
Nevermind - I see that you got it.